### PR TITLE
Typos and info updated

### PR DESCRIPTION
**Pull request by jcarolinares**
## What This PR Changes
- Typos and info updated

## Contribution Guidelines
- [ x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/05.pro-solutions/solutions-and-kits/opta/tech-specs.yml [hunk 1]
```diff
@@ -34,8 +34,8 @@ Dimensions:
   Length: 80 mm
   Height: 90 mm
 IP Protection: IP20
-RTC: ~10days, NTP sync through ethernet
+RTC: ~10days, NTP sync through Ethernet or Wifi
```
Comment (aliphys) [inline content/hardware/05.pro-solutions/solutions-and-kits/opta/tech-specs.yml:37]: Would it be more clear to state as follows:
```suggestion
RTC: ~10days, NTP sync through Wi-Fi® (AFX00002 only) or Ethernet
```

File: content/hardware/05.pro-solutions/solutions-and-kits/opta/tech-specs.yml [hunk 2]
```diff
@@ -34,8 +34,8 @@ Dimensions:
...
 Secure element: ATECC608B
 Programming:
   Arduino programming language: Via Arduino IDEs, Arduino CLI, Arduino Web Editor
-  IEC-61131-3 as option: Ladder Diagram (LD), Function Block Diagram (FBD), Sequential Function Chart (SFC), Structured Text (ST), Instruction List (IL)
+  IEC-61131-3: Ladder Diagram (LD), Function Block Diagram (FBD), Sequential Function Chart (SFC), Structured Text (ST), Instruction List (IL)
```
Comment (aliphys) [inline content/hardware/05.pro-solutions/solutions-and-kits/opta/tech-specs.yml:41]: This is only for the Arduino PLC IDE right?
```suggestion
  IEC-61131-3 via Arduino PLC IDE: Ladder Diagram (LD), Function Block Diagram (FBD), Sequential Function Chart (SFC), Structured Text (ST), Instruction List (IL)
```

File: content/hardware/05.pro-solutions/solutions-and-kits/opta/tech-specs.yml (hunk 1 above)
Comment (jcarolinares) [reply content/hardware/05.pro-solutions/solutions-and-kits/opta/tech-specs.yml:37]: I like it, accepted, thanks!

File: content/hardware/05.pro-solutions/solutions-and-kits/opta/tech-specs.yml (hunk 2 above)
Comment (jcarolinares) [reply content/hardware/05.pro-solutions/solutions-and-kits/opta/tech-specs.yml:41]: Let's keep as it is just in case, have my reasons :)

**Review by aliphys — APPROVED** [review]
LGTM

**Pull request merged**